deploy/helm/flightctl/templates/flightctl-preupgrade-scale-jobs.yaml

```
@@ -0,0 +1,131 @@
+{{- /* Determine whether to render the pre-upgrade scale job */ -}}
+{{- $allDeployments := default (list) .Values.upgradeHooks.scaleDown.deployments -}}
+{{- $condition := default "chart" .Values.upgradeHooks.scaleDown.condition -}}
+{{- $deploymentsToScale := list -}}
+
+{{- if gt (len $allDeployments) 0 -}}
+  {{- if eq $condition "always" -}}
+    {{- $deploymentsToScale = $allDeployments -}}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 0 but found 4

(indentation)